can_tools/scrapers/official/VA/va_vaccine.py

```diff
@@ -81,3 +78,288 @@ def normalize(self, data) -> pd.DataFrame:
+        df["dt"] = self._retrieve_dt()
+        df = df.rename(
+            columns={
+                "AGG(Locality Name for String)-alias": "location_name",
+                "SUM(Vaccine Status Count)-value": "value",
+                "ATTR(KPI String)-alias": "category",
+                "Sex-alias": "sex",
+            }
+        )
+        df.category = df.category.str.replace(
+            "People Fully Vaccinated", "total_vaccine_completed"
+        )
+        df.category = df.category.str.replace(
+            "People with At Least One Dose", "total_vaccine_initiated"
+        )
+        df["age"] = "all"
+        df["race"] = "all"
+        df["ethnicity"] = "all"
+        df.sex = df.sex.str.lower()
+        return df[
+            [
+                "dt",
+                "location_name",
+                "category",
+                "sex",
+                "race",
+                "ethnicity",
+                "age",
+                "value",
+            ]
+        ]
+
+    def _normalize_age(self, data):
+        df = pd.concat(data["age"])
+        df["dt"] = self._retrieve_dt()
+        df = df.rename(
+            columns={
+                "AGG(Locality Name for String)-alias": "location_name",
+                "SUM(Vaccine Status Count)-value": "value",
+                "ATTR(KPI String)-alias": "category",
+                "Age Group-alias": "age",
+            }
+        )
+        df.category = df.category.str.replace(
+            "People Fully Vaccinated", "total_vaccine_completed"
+        )
+        df.category = df.category.str.replace(
+            "People with At Least One Dose", "total_vaccine_initiated"
+        )
+        df.age = df.age.str.replace("+", "_plus")
+        df["race"] = "all"
+        df["sex"] = "all"
+        df["ethnicity"] = "all"
+        return df[
+            [
+                "dt",
+                "location_name",
+                "category",
+                "sex",
+                "race",
+                "ethnicity",
+                "age",
+                "value",
+            ]
+        ]
+
+    def _normalize_race(self, data):
+        df = pd.concat(data["race"])
+        df["dt"] = self._retrieve_dt()
+        df = df.rename(
+            columns={
+                "AGG(Locality Name for String)-alias": "location_name",
+                "SUM(Vaccine Status Count)-value": "value",
+                "ATTR(KPI String)-alias": "category",
+                "Race and Ethnicity-alias": "race",
+            }
+        )
+        df.category = df.category.str.replace(
+            "People Fully Vaccinated", "total_vaccine_completed"
+        )
+        df.category = df.category.str.replace(
+            "People with At Least One Dose", "total_vaccine_initiated"
+        )
+
+        df.race = df.race.str.replace("White", "white")
+        df.race = df.race.str.replace("Other Race", "other")
+        df.race = df.race.str.replace("Native American", "native_american")
```

Comment:
It looks like we have both `ai_an` (American Indian or Alaska Native) and `native_american` in our demographics schema. Is this intended? I've seen Native American categorized as `ai_an` in the past (in our scrapers), but I don't know if there's a distinction.

Reply:
I put it as just Native American because that's how the data had it...but maybe just putting our existing `ai_an` will be better. I'll change it

Reply:
That's fair--I ask in large part because I'm not sure myself